### ISSUE TITLE
<html> element does not have a [lang] attribute

### ISSUE THREAD

**Issue by bertysentry** [sentrysoftware/sentry-maven-skin#229]
## Problem

`<html>` element does not have a [lang] attribute
If a page doesn't specify a lang attribute, a screen reader assumes that the page is in the default language that the user chose when setting up the screen reader. If the page isn't actually in the default language, then the screen reader might not announce the page's text correctly. [Learn more about the lang attribute](https://dequeuniversity.com/rules/axe/4.11/html-has-lang).

## Specification

Add the `lang` attribute to the `<html>` element, and make sure it matches with the $locale being rendered.

If $locale is the default, we need to determine the actual locale (probably "English", but it could be theoretically different).

From [MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Global_attributes/lang):

> The lang [global attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Global_attributes) helps define the language of an element: the language that non-editable elements are written in, or the language that the editable elements should be written in by the user. The attribute contains a single [BCP 47 language tag](https://developer.mozilla.org/en-US/docs/Glossary/BCP_47_language_tag).

### BCP-47

The full BCP 47 syntax is specified in [RFC 5646](https://datatracker.ietf.org/doc/html/rfc5646). It is capable of identifying extremely specific language dialects, but most usage is much simpler.

A language tag is made of hyphen-separated subtags, where each subtag indicates a certain property of the language. The three most common subtags are:

[Language subtag](https://developer.mozilla.org/en-US/docs/Glossary/BCP_47_language_tag#language_subtag)
A 2-or-3-character code that defines the basic language, typically written in all lowercase. For example, the language code for English is en, and the code for Badeshi is bdz.

[Script subtag Optional](https://developer.mozilla.org/en-US/docs/Glossary/BCP_47_language_tag#script_subtag_optional)
This subtag defines the writing system used for the language, and is always 4 characters long, with the first letter capitalized. For example, French-in-Braille is fr-Brai and Japanese written with the Katakana alphabet is ja-Kana.

Note: If the language is written in a highly typical way, like English in the Latin alphabet, there is no need to use this subtag.

[Region subtag Optional](https://developer.mozilla.org/en-US/docs/Glossary/BCP_47_language_tag#region_subtag_optional)
This subtag defines a dialect of the base language from a particular location and is either two upper-case letters matching a country code or three numbers matching a non-country area. For example, es-ES is for Spanish as spoken in Spain, and es-013 is Spanish as spoken in Central America. "International Spanish" would just be es.

The script subtag precedes the region subtag if both are present — ru-Cyrl-BY is Russian, written in the Cyrillic alphabet, as spoken in Belarus.
